categories_map.py

```diff
@@ -541,7 +542,6 @@
     "DQM/SiPixelPhase1Config",
     "DQM/SiPixelPhase1DeadFEDChannels",
     "DQM/SiPixelPhase1Digis",
-    "DQM/SiPixelPhase1Heterogeneous",
```

Comment:
The earlier package needs to be kept (the list should effectively be a union of all CMSSW releases).